sdks/python/src/honcho/async_client/session.py

```
@@ -468,6 +469,54 @@ async def search(
         )
         return AsyncPage(messages_page)
 
+    @validate_call
+    async def upload_file(
+        self,
+        file: tuple[str, bytes, str] | tuple[str, Any, str] | Any = Field(
```

Ruff (0.12.2)

Dynamically typed expressions (typing.Any) are disallowed in `file`

(ANN401)